Detectors/Base/include/DetectorsBase/MatCell.h

```diff
@@ -30,7 +30,7 @@ struct MatCell {
   float meanX2X0;                   ///< fraction of radiaton lenght
 
   MatCell() : meanRho(0.f), meanX2X0(0.f) {}
-  MatCell(const MatCell& src) : meanRho(src.meanRho), meanX2X0(src.meanX2X0) {}
+  MatCell(const MatCell& src) CON_DEFAULT;
```

Comment:
is CON_DEFAULT a macro abstracting on `= default`? Do we need it since this code is GPU compiled?